sys-apps/spiff/spiff-1.recipe

```diff
@@ -1,58 +1,50 @@
-DESCRIPTION="
-'spiff' is a command line program that compares the contents of two files and \
-prints a description of the important differences between them in a form \
-rather like the standard 'diff' utility.  However, unlike diff, it is not line \
-oriented -- it compares word sequences with all white space and newlines \
-disregarded, so even if formatting is different you only see the significant \
-differences.  It also compares floating-point values with a tolerance, so you \
-can look in data files for discrepancies.
-"
 SUMMARY="Advanced command line tool like standard 'diff'"
+DESCRIPTION="spiff is a command line program that compares the contents of \
+two files and prints a description of the important differences between them \
+in a form rather like the standard 'diff' utility.  However, unlike diff, it \
+is not line oriented -- it compares word sequences with all white space and \
+newlines disregarded, so even if formatting is different you only see the \
+significant differences.  It also compares floating-point values with a \
+tolerance, so you can look in data files for discrepancies."
 HOMEPAGE="https://github.com/HaikuArchives/Spiff"
-SOURCE_URI="git+https://github.com/HaikuArchives/Spiff#50fa5e3cfe1431071b1de57828508f6903b052c2"
+COPYRIGHT="1988 Bellcore"
+LICENSE="SPIFF"
 REVISION="1"
-#STATUS_HAIKU="stable"
-ARCHITECTURES="x86_gcc2 ?x86 ?x86_64"
+SOURCE_URI="https://github.com/HaikuArchives/Spiff/archive/master.zip"
```

Comment:
Yoou should use a git revision instead of "master" because it doesn't define clearly, which version is it, and you need to fix the checksum after every commit.